spatial-history/index.md

```diff
@@ -30,10 +13,23 @@ It makes sense to put an introductory paragraph here, above the map and buttons.
   <div id="map_legend"></div>
 </div>
 
-# Next Section
+## **Before the Landing**
+
+The five beaches marked along the Normandy Coastline were Gold, Sword, Juno, Omaha and Utah beach. Gold and Sword beach’s were attacked by the British, Juno by the Canadians leaving the Americans with Omaha and Utah beaches. Consider the planning involved with such an operation and the exact detailing allied intelligence had to do when building maps for the landing. In total, 156,000 soldiers were landed on the five beaches which spanned approximately 100km [Richard Foot](http://www.thecanadianencyclopedia.ca/en/article/normandy-invasion/). Furthermore, on top of the landing forces, British, Canadian and US paratroopers landed behind enemy lines with the specific task of disabling key communication [Ibid](http://www.thecanadianencyclopedia.ca/en/article/normandy-invasion/). Why is this important? Well, not only did Allied intelligence need to create a detailed map for their own spatial awareness they also needed to transfer this knowledge to the soldiers trying to land, “As soldiers waited in their camps, they pored over maps, photographs and three-dimensional models of the invasion beaches, constructed from information meticulously collected over the previous months. The models and pictures showed the layout of the Normandy coastline and important landmarks — houses, church spires, headlands — so that every officer and soldier would know their objectives and what awaited them”[Ibid](http://www.thecanadianencyclopedia.ca/en/article/normandy-invasion/).
```

Comment:
small improvements to make:
- set citations off from the main text with parentheses or some other visual marker
- It's true that D-Day is general knowledge but use of some scholarly sources would be good here. 
- the main point here is significant, but maybe something about why the terrain/spatial awareness was important -- that's a hidden asumption here.